### PR TITLE
data: mkbhru

### PR DESCRIPTION
**Pull request by mkbhru**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4728"></a>

### CODE REVIEW

File: data/mkbhru.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mkbhru",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/mkbhru",
+      "icon": "FaGithubAlt"
+    },
```
Comment (swagataroy30) [inline data/mkbhru.json:16]:
```suggestion
    }
```

**Review by swagataroy30 — COMMENTED** [review]
Can you please remove the comma from line 16. Thank You!

**Review by Sindhuinti — APPROVED** [review]
Looks good to me. Thanks 👍🏻

**Review by iamthecloverly — APPROVED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

File: data/mkbhru.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Manish Kumar",
+  "type": "personal",
+  "bio": "A fullstack developer",
+
```
Comment (krshkun) [inline data/mkbhru.json:5]: Looks great, this empty line will be fixed by github actions 👍

**Review by krshkun — APPROVED** [review]

File: data/mkbhru.json (hunk 2 above)
Comment (mkbhru) [reply data/mkbhru.json:5]: thanks for accepting it :+1:

**Comment by mkbhru** [comment]
Hey folks, Now what I do with this opened request should I close it???

**Comment by krshkun** [comment]
When a maintainer checks it out, they will merge it in 👍

**Review by Hrishi5111998 — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**